src/complain.c

```diff
@@ -113,6 +113,7 @@ static const argmatch_warning_doc argmatch_warning_docs[] =
 {
   { "conflicts-sr",     N_("S/R conflicts (enabled by default)") },
   { "conflicts-rr",     N_("R/R conflicts (enabled by default)") },
+  { "counterexample",   N_("Conflict counter examples") },
```

Comment:
We should stick to "counterexample" everywhere.
Can be addressed after the merge.